src/senaite/core/adapters/configure.zcml

```diff
@@ -38,13 +38,14 @@
   <!-- Dynamic calculation of local roles for a given context and principal -->
   <adapter factory=".localroles.DynamicLocalRoleAdapter" />
 
+  <!-- Superseded by Client Group which has the right permission -->
   <!-- Dynamic assignment of local roles for objects that belong to the same
   Client as the contact the current user is linked to -->
-  <adapter
-      for="*"
-      provides="senaite.core.interfaces.IDynamicLocalRoles"
-      factory=".localroles.ClientAwareLocalRoles"
-      name="senaite.core.adapter.localroles.clientaware" />
+  <!-- <adapter -->
+  <!--     for="*" -->
+  <!--     provides="senaite.core.interfaces.IDynamicLocalRoles" -->
+  <!--     factory=".localroles.ClientAwareLocalRoles" -->
+  <!--     name="senaite.core.adapter.localroles.clientaware" /> -->
```

Comment:
I've done a quick search (extensions included) and `ClientAwareLocalRoles` is not used anywhere else. I suggest to remove (instead of comment) this adapter declaration, as well as the adapter itself (`senaite.core.adapters.localroles.ClientAwareLocalRoles`)

Reply:
Removed in d95395833